src/main/java/org/tndata/android/compass/adapter/feed/MainFeedAdapter.java

```diff
@@ -225,6 +233,10 @@ public void onBindViewHolder(RecyclerView.ViewHolder rawHolder, int position){
         else if (CardTypes.isUpNext(position)){
             ((UpNextHolder)rawHolder).bind(mFeedData.getUpNextAction(), mFeedData.getProgress());
         }
+        //Streaks
+        else if (CardTypes.isStreaks(position)){
+            ((StreaksHolder)rawHolder).bind(mFeedData.getStreaks());
```

Comment:
This bind method gets called every time the data-set is updated (for all intents and purposes). https://github.com/tndatacommons/android-app/pull/215/files#r71026170